checkbox-core-snap/series22/snap/snapcraft.yaml

```diff
@@ -438,6 +442,18 @@ parts:
       python3 manage.py build
       python3 manage.py install --layout=relocatable --prefix=/providers/checkbox-provider-certification-server --root="$SNAPCRAFT_PART_INSTALL"
     after: [checkbox-provider-certification-client]
+################################################################################
+  checkbox-provider-tutorial:
```

Comment:
Interesting. I haven't seen the value in *delivering* the tutorial provider with the rest of the stack. I thought it's more of a development thing that's meant to be run venv.

Reply:
One of the first sections of the tutorial, after installing Checkbox via snaps, is about running the tutorial test plan:

https://checkbox.readthedocs.io/en/latest/tutorial/using-checkbox/basic-running.html

So it's required (or the base tutorial needs to be re-written :))

Reply:
Right. Then it makes total sense :)